k9mail-library/src/main/java/com/fsck/k9/mail/internet/MimeUtility.java

```diff
@@ -1153,4 +1155,12 @@ static boolean isFormatFlowed(String contentType) {
         }
         return false;
     }
+
+    static boolean isDelSp(String contentType){
```

Comment:
I don't see a reason to have such a specialized method in `MimeUtility`. This should probably live in `FlowedMessageUtils`.

Also, please adhere to our code style. There are some missing spaces and indentation errors.